src/CopyContentButton.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import $ from "jquery";
+import { copyFormatted } from "./copy-to-clipboard";
+export default class CopyContentButton extends React.Component {
+    handleCopy = event => {
+      const text = this.getFormattedSignature(
+        this.props.userData.fullName,
+        this.props.userData.position.label,
+        this.props.userData.title,
+        this.props.userData.email,
+        this.props.userData.skypeId
+      );
+      copyFormatted(text);
+      $(".fixed")
+        .show(1)
+        .delay(1000)
+        .hide(1);
+    };
+  
+    render() {
+      return (
+        <button className="footer-small-cage"
+                  onClick={this.handleCopy}>
+                  Copy Info
+                </button>
+      )
+    }
+  
+    getFormattedSignature(fullName, position, str, email, skypeId) {
+      return `${fullName}<br/>
+  ${position}</br>
+  ${str.bold()}</br>
+  <a href="mailto:${email}">${email} </a></br>
```

Comment:
Is bootstrap available? If not, can we use something like bootstrap to use divs instead of </br> in template.